infrastructure/src/main/java/org/apache/geode/infrastructure/aws/LaunchCluster.java

```diff
@@ -241,19 +241,21 @@ private static void createKeyPair(String benchmarkTag) throws IOException {
   private static void createMetadata(String benchmarkTag, List<String> publicIps)
       throws IOException {
     UUID instanceId = UUID.randomUUID();
-    JSONObject metadataJSON = new JSONObject();
-
-    metadataJSON.put("instanceId", instanceId.toString());
-    metadataJSON.put("publicIps", new JSONArray(publicIps));
+    // TODO - Filter out only benchmark properties from system properties? Maybe not necessary.
```

Comment:
Please cleanup TODO.

Reply:
Done! Thanks for catching that. I just removed the TODO. This file is just metadata to look at later to see the entire test configuration. , So it's actually not a bad thing that it's dumping all the system properties and not just the benchmark ones.